app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/completeregistration/CompleteRegistrationScreen.kt

```diff
@@ -135,11 +136,24 @@ fun CompleteRegistrationScreen(
             .nestedScroll(scrollBehavior.nestedScrollConnection),
         topBar = {
             BitwardenTopAppBar(
-                title = stringResource(id = R.string.create_account),
+                title = if (state.onBoardingEnabled) {
```

Comment:
⛏️ I think it should be `onboarding`, not `on boarding`. Probably out of scope to correct in this PR. 🤷

Reply:
not a problem!